src/schema/schema.graphql

```diff
@@ -1140,6 +1140,7 @@ type Folder {
 	broken: Boolean #shared folder link is broken or not
 	deletable: Boolean
 	unreadDescendent: Boolean
+	types: String
```

Comment:
this should be converted to an enum with valid values
check https://files.zimbra.com/docs/soap_api/9.0.0/api-reference/zimbraMail/GetFolder.html

Reply:
for mail vertical type can go as "". and enum doesn't accept empty string. Getting error like this. index.js:207 Error: Enum type FolderView cannot include value: "".
Tried with null value as well. Getting same error.